confluent_kafka/__init__.py

```diff
@@ -634,3 +634,26 @@ def __repr__(self):
 
     def __str__(self):
         return "{}".format(self.partition)
+
+
+class ThrottleEvent (object):
+    """
+    ThrottleEvent passed as an argument to throttle_cb.
+    Contains details about requests throttled by the broker.
```

Comment:
It contains information pertaining to one single throttled request, don't know how to formulate that properly.